### PR TITLE
Update structure of Art Reg ticket with new rareness scores.

### PR DESCRIPTION
**Pull request by danileng20**
Update Art Reg structure with new changes in Rareness scores and other info as followings:
"fingerprints" []float
"dupe_detection_system_version" string
"matches_found_on_first_page" int
"number_of_pages_of_results" int
"url_of_first_match_in_page" string
"pastel_rareness_score" float
"internet_rareness_score" float
"open_nsfw_score" float
"alternate_nsfw_scores" object
"image_hashes" object

alternate NSFW scores object:
"drawings" float
"hentai" float
"neutral" float
"porn" float
"sexy" float

image hashes object:
"perceptual_hash" string
"average_hash" string
"average_hash" string

### CODE REVIEW

**Review by pjenys0 — DISMISSED** [review]
LGTM!

**Review by pjenys0 — APPROVED** [review]

File: pastel/reg_ticket.go [hunk 1]
```diff
@@ -54,18 +71,155 @@ type AppTicket struct {
...
 
-	ThumbnailHash []byte `json:"thumbnail_hash"`
-	DataHash      []byte `json:"data_hash"`
+	PreviewHash    []byte `json:"preview_hash"`
+	Thumbnail1Hash []byte `json:"thumbnail1_hash"`
+	Thumbnail2Hash []byte `json:"thumbnail2_hash"`
+
+	DataHash []byte `json:"data_hash"`
+
+	Fingerprints          []float64 `json:"fingerprints"`
```
Comment (leonidcode20) [inline pastel/reg_ticket.go:80]: as comment from ayk then `Fingerprints` should be removed ?

Comment (danileng20) [reply pastel/reg_ticket.go:80]: no, fingerprints now is an array of floats

**Review by leonidcode20 — APPROVED** [review]
LGTM!

**Pull request merged**